python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -907,6 +912,9 @@ def execute_asset_backfill_iteration(
         workspace_context, backfill, asset_graph, instance_queryer, logger
     )
     if previous_asset_backfill_data is None:
+        tick_context.update_state(
+            TickStatus.SUCCESS
+        )  # should this be a fail? it's bc the code location is unavailable?
```

Comment:
`previous_asset_backfill_data` will be None if the code location is unloadable and the user has an env var set to not raise an error when the code location is unloadable. To me, this indicates that the user does not want to consider an unloadable code location as an error, therefor the tick should be marked success